### PR TITLE
add global styles and font

### PR DESCRIPTION
**Pull request by cmaiw**

### CODE REVIEW

File: src/GlobalStyles.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          flex-direction: column;
+          height: 100vh;
+          width: 100vw;
+          justify-content: center;
+          align-items: center;
+          margin: 0;
+          padding:0
+          background-color: navy;
+          color: whitesmoke;
+          font-family: font-family: 'Odibee Sans', cursive;;
```
Comment (lmachens) [inline src/GlobalStyles.js]: duplicate `font-family`

File: src/index.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import styled from "@emotion/styled";
+import GlobalStyles from "./GlobalStyles";
```
Comment (lmachens) [inline src/index.js]: You should import `GlobalStyles` in your `App.js` and use it there

File: src/GlobalStyles.js (hunk 1 above)
Comment (cmaiw) [reply src/GlobalStyles.js]: thank you.

**Comment by cmaiw** [comment]
> I would suggest to import GlobalStyles in App.js maybe!


> I would suggest to import GlobalStyles in App.js maybe!

Thank you.

**Pull request merged**